### ISSUE TITLE
Problem z ograniczaniem ilości zapytań API

### ISSUE THREAD

**Issue by SystemNetLeaders** [allegro/allegro-api#9507]
Występuje problem: 
 - [X] z zasobem REST API / with a REST API resource
 - [ ] ze środowiskiem produkcyjnym / with the production environment
 - [ ] ze środowiskiem testowym / with the test environment
 - [ ] z logowaniem lub autoryzacją / with login or authorization
 - [ ] na portalu https://developer.allegro.pl/ / at https://developer.allegro.pl/en/
 - [ ] inny / other

:beetle: **Opis / Description**

Na sklepie WooCommerce posiadamy integrację, która łączy się poprzez wtyczkę WP Desk z Allegro API. Mamy obecnie niecałe 7000 produktów zaimportowanych. Od kilku dni sklep zaczął strasznie wolno chodzić - zauważyliśmy, że problem może być po stronie API Allegro. Czy istnieje jakiś limit zapytań albo coś innego co mogłoby spowolnić działanie API?

cURL: https://danielautoparts.com/wp-admin/admin.php?page=woocommerce_allegro_settings&allegro_auth_code=1&profile_id=66826a6d03921

**Comment by AureliuszBrussy** [comment]
Tak, limit zapytań wynosi 9000 req/min. Dodatkowo, niektóre z endpointów mają dodatkowe limity - o nich informujemy [w dokumentacji](https://developer.allegro.pl/documentation), przy opisie danego endpointu. Jeżeli trafiacie na limit - zwracamy status code 429.
